tests/DB/QueryBuilderTest.php

```diff
@@ -177,6 +191,20 @@ public function testOrderByMulti()
 		$this->assertEquals($expected, trim(preg_replace('/\s+/', ' ', $query)));
 	}
 
+	public function testOrderByArray()
+	{ 
+		$query = $this->_builder
+			->select("*")
+			->from('table')
+			->orderBy(['col', 'col2'])
+			->getQueryString()
+		;
+
+		$expected = "SELECT * FROM table ORDER BY col, col2";
+
+		$this->assertEquals($expected, trim(preg_replace('/\s+/', ' ', $query)));
+	}
+
 	public function testLimit()
```

Comment:
We have them for https://github.com/mothership-ec/cog/pull/453#discussion-diff-34275223R179 regular maybe we should add them for the array ones. Also maybe we should call orderBy with arrays and without, mix it up.